src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -49,37 +57,75 @@ const Dokumenter: React.FC<{ personopplysninger: IPersonopplysninger }> = ({
-                                                            >
-                                                                {dokument.tittel}
-                                                            </Lenke>
-                                                        </Td>
-                                                    </tr>
+                                        {Object.keys(grupperteDokumenter).map(
+                                            (journalpostId: string) => {
+                                                return grupperteDokumenter[journalpostId].map(
+                                                    (dokument: Dokumentinfo, indeks: number) => {
+                                                        if (indeks === 0) {
+                                                            return (
+                                                                <TrHoveddokument key={indeks}>
+                                                                    <Td>
+                                                                        {formaterNullableIsoDatoTid(
+                                                                            dokument.dato
+                                                                        )}
+                                                                    </Td>
+                                                                    <Td>
+                                                                        <Lenke
+                                                                            onClick={() =>
+                                                                                hentDokument(
+                                                                                    dokument
+                                                                                )
+                                                                            }
+                                                                            href={'#'}
+                                                                        >
+                                                                            {dokument.tittel}
+                                                                        </Lenke>
+                                                                    </Td>
+                                                                    <Td>
+                                                                        <Normaltekst>
+                                                                            {dokument.journalstatus}
+                                                                        </Normaltekst>
+                                                                    </Td>
+                                                                </TrHoveddokument>
+                                                            );
```

Comment:
Kanskje man kan skrive om <TrHoveddokument key={indeks}> til en egen komponent for økt lesbarhet?

Reply:
Yes, fikset i 06b8d38